src/components/picture/index.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Picture from ".";
+import Source from "./_children/source";
+import Image from "../image";
+
+<Meta title="Components/Picture" component={Picture} />
+
+# Picture
+
+The `Picture` component and `Picture.Source` subcomponent render the HTML5 `<picture>` and `<source>` tags, respectively. Similar to the Image, the Picture component displays images as both a means of content and content support. Picture, however, allows for different images (with respective resize options) to be used explicitely for different media queries. This allows for different Art Direction approaches for specific viewport dimensions.
+
+For `resizedOptions`, here are the query parameters that are available:
+https://github.com/WPMedia/arc-photo-resizer/tree/main/docs/usage#query-parameters
+
+Crucially, an "auth" key will allow the image to be resized. This object will resize the image to the specified dimensions and other properties given in the query parameters in the `src`.
+
+The `Picture` component can use zero or more `Picture.Source` subcomponents and should also include one `Image` as a supportive fallback. Those are the two types of children a Picture will support.
```

Comment:
Not sure why someone would use the picture component but not a picture.source. Not sure we'd want to encourage zero
```suggestion
The `Picture` component can take in `Picture.Source` subcomponents and should also include one `Image` as a supportive fallback. Those are the two types of children a Picture will support.
```

Reply:
It's per standard.  A `picture` must have one `img` and zero or more `source`s.

https://html.spec.whatwg.org/multipage/embedded-content.html#the-picture-element

![image](https://user-images.githubusercontent.com/2287238/182227552-02d12929-c751-4574-ac73-f200467bbd47.png)

Reply:
cool no worries